src/server/src/main/java/io/cassandrareaper/service/RepairRunner.java

```diff
@@ -167,14 +169,20 @@ UUID getRepairRunId() {
 
   static int getPossibleParallelRepairsCount(
       Map<List<String>, List<String>> ranges,
-      Map<String, String> hostsInRing) throws ReaperException {
+      Map<String, String> hostsInRing,
+      DatacenterAvailability datacenterAvailability) throws ReaperException {
```

Comment:
trivial: could the method could have just been made non-static? (not sure how much change that would require in the tests, but i suspect not much…)

Reply:
Actually, after giving it a try, it involves a lot of additional code to mock stuff. Instantiating a RepairRunner triggers a lot of calls to dependencies.